eng/common/mcp/README.md

```diff
@@ -0,0 +1,38 @@
+# Azure SDK MCP Servers
+
+This document details how to author, publish and use [MCP servers](https://github.com/modelcontextprotocol) for azure sdk team usage.
+
+## Using the Azure SDK MCP Server
+
+Run the below command to download and run the azure sdk engsys mcp server manually:
+
+```
+<repo root>/eng/common/mcp/azure-sdk-mcp.ps1 -Run
+```
+
+To install the mcp server for use within vscode copilot agent mode, run the following then launch vscode from the repository root.
+
+```
+<repo root>/eng/common/mcp/azure-sdk-mcp.ps1 -UpdateVsCodeConfig
+```
+
+*When updating the config the script will not overwrite any other server configs.*
+
+The script will install the latest version of the azsdk cli executable from [tools releases](https://github.com/Azure/azure-sdk-tools/releases) and install it to `$HOME/.azure-sdk-mcp/azsdk`.
```

Comment:
Do we really want to put this into the repo? I know you added it to the ignore for tools but that will mean we need to do it for any consuming repos as well.

Reply:
Nevermind HOME is not in the repo, I was through off by you adding it to the tools ignore path. Why did you need to add that?

Reply:
Ah, bad carry over from when I had it in the repo root originally. I'll remove.